catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/Nsili.java

```diff
@@ -24,6 +25,7 @@
 public interface Nsili {
 
     @GET
-    @Produces(MediaType.WILDCARD)
+    @Consumes({MediaType.TEXT_PLAIN, MediaType.APPLICATION_XML, MediaType.APPLICATION_OCTET_STREAM, MediaType.WILDCARD})
```

Comment:
If we have to explicitly add all these, what is `MediaType.WILDCARD` doing exactly?